fem/fe.hpp

```diff
@@ -513,6 +513,12 @@ class FiniteElement
    virtual void Project (VectorCoefficient &vc,
                          ElementTransformation &Trans, Vector &dofs) const;
 
+   /** @brief Given a vector of values at the finite element nodes and a transformation,
+       compute its projection (approximation) in the local finite dimensional space
+       in terms of the degrees of freedom. (VectorFiniteElements) */
+   virtual void ProjectFromElementNodes (Vector &vc,
+                                         ElementTransformation &Trans, Vector &dofs) const;
```

Comment:
I think `ProjectFromNodes` is a better name, as all methods in this class work on local level.